.distinclude

```diff
@@ -1,17 +1,21 @@
 # List all files and directories to be deployed with the Go thene.
```

Comment:
Is it a copy-paste error of talking about Go or it's talking about the Go install that CoBlocks is using?
Also, it should say `theme`

Reply:
Yeah, you are right. This should just be "deployed with CoBlocks". I've updated that, good catch 👍